src/components/Provider.js

```diff
@@ -51,7 +51,14 @@ const getOrCreateStripe = (apiKey: string, options: mixed): StripeShape => {
 };
 
 const ensureStripeShape = (stripe: mixed): StripeShape => {
-  if (stripe && stripe.elements && stripe.createSource && stripe.createToken) {
+  if (
+    stripe &&
+    stripe.elements &&
+    stripe.createSource &&
+    stripe.createToken &&
+    stripe.createPaymentMethod &&
+    stripe.handleCardPayment
```

Comment:
Hmm, this gets a little awkward. What if people pass in a Stripe instance that they instantiated themselves with one of the old PI beta flags that doesn't provide `createPaymentMethod`? Should we just say "this new major version only supports the unbeta'd state?" I think that's an ok decision, just want to check that it's deliberate.

Reply:
I think we have to make a major version cut that makes sense.